plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOraclePoolConnectorSmokeTest.java

```diff
@@ -27,16 +26,16 @@
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TestOraclePoolConnectorTest
-        extends BaseOracleConnectorTest
+public class TestRemarksReportingOraclePoolConnectorSmokeTest
+        extends BaseOracleConnectorSmokeTest
 {
     private TestingOracleServer oracleServer;
```

Comment:
Did you mean to remove this field too?

Reply:
i did  & reverted to keep diff smaller. did i remove elsewhere?

Reply:
The commit I looked at removed the assignment to the local var so it looked like it was unused (maybe I didn't look at the `destroy` method).

It is correct. Consider resolved.